src/element/mod.rs

```diff
@@ -256,13 +256,16 @@ impl IonEq for Element {
 /// An `(annotations, value)` pair representing an Ion value.
 #[derive(Debug, Clone)]
 pub struct Element {
-    annotations: Vec<Symbol>,
+    annotations: Annotations,
     value: Value,
 }
 
 impl Element {
-    pub fn new(annotations: Vec<Symbol>, value: Value) -> Self {
-        Self { annotations, value }
+    pub(crate) fn new(annotations: Annotations, value: impl Into<Value>) -> Self {
```

Comment:
🗺️ I've restricted access to the constructors for `Element` and `Annotations` to encourage users to reach for the more flexible builders and constructor methods. We can reintroduce it later if needed.